28/02/2018 16:01:54,934==>Traza error (interno): Error java.lang.NullPointerException:co.com.codesa.socketcodesaoperadorexterno.dao.pool.ConnectionPool.cerrarConexion(ConnectionPool.java:139)|co.com.codesa.socketcodesaoperadorexterno.dao.pool.ConnectionPool.getDiaActualBD(ConnectionPool.java:520)|co.com.codesa.socketcodesaoperadorexterno.dao.pool.hilo.HiloCambioDia.<init>(HiloCambioDia.java:27)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:86)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$1.run(EventoFormaConfiguracion.java:73)|java.lang.Thread.run(Thread.java:745)|
28/02/2018 16:04:08,550==>Traza error (interno): Error java.lang.NullPointerException:co.com.codesa.socketcodesaoperadorexterno.dao.pool.ConnectionPool.cerrarConexion(ConnectionPool.java:139)|co.com.codesa.socketcodesaoperadorexterno.dao.pool.ConnectionPool.getDiaActualBD(ConnectionPool.java:520)|co.com.codesa.socketcodesaoperadorexterno.dao.pool.hilo.HiloCambioDia.<init>(HiloCambioDia.java:27)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:86)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$1.run(EventoFormaConfiguracion.java:73)|java.lang.Thread.run(Thread.java:745)|
28/02/2018 16:18:05,990==>Traza error (interno): Error java.lang.NullPointerException:co.com.codesa.socketcodesaoperadorexterno.dao.pool.ConnectionPool.cerrarConexion(ConnectionPool.java:139)|co.com.codesa.socketcodesaoperadorexterno.dao.pool.ConnectionPool.getDiaActualBD(ConnectionPool.java:520)|co.com.codesa.socketcodesaoperadorexterno.dao.pool.hilo.HiloCambioDia.<init>(HiloCambioDia.java:27)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:86)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$1.run(EventoFormaConfiguracion.java:73)|java.lang.Thread.run(Thread.java:745)|
25/08/2018 10:50:32,380==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
25/08/2018 11:37:32,664==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
04/09/2018 11:22:43,568==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
14/09/2018 14:33:25,284==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
14/09/2018 14:45:45,251==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
18/09/2018 09:10:33,837==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
18/09/2018 09:32:04,946==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
21/09/2018 16:09:53,606==>Traza error (interno): Error java.net.BindException: La dirección ya se está usando:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:722)|
21/10/2018 21:13:09,470==>Traza error (interno): Error java.net.BindException: Address already in use: JVM_Bind:java.net.DualStackPlainSocketImpl.bind0(Native Method)|java.net.DualStackPlainSocketImpl.socketBind(DualStackPlainSocketImpl.java:106)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.PlainSocketImpl.bind(PlainSocketImpl.java:190)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:745)|
23/10/2018 09:58:37,556==>Traza error (interno): Error java.net.BindException: Address already in use: JVM_Bind:java.net.DualStackPlainSocketImpl.bind0(Native Method)|java.net.DualStackPlainSocketImpl.socketBind(DualStackPlainSocketImpl.java:106)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.PlainSocketImpl.bind(PlainSocketImpl.java:190)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$2.run(EventoFormaConfiguracion.java:142)|java.lang.Thread.run(Thread.java:745)|
01/10/2019 14:29:37,273==>Traza error (interno): Error java.net.BindException: Address already in use:java.net.PlainSocketImpl.socketBind(Native Method)|java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:376)|java.net.ServerSocket.bind(ServerSocket.java:376)|java.net.ServerSocket.<init>(ServerSocket.java:237)|java.net.ServerSocket.<init>(ServerSocket.java:181)|co.com.codesa.socketcodesaoperadorexterno.socket.ServerWS.subirServidor(ServerWS.java:48)|co.com.codesa.socketcodesaoperadorexterno.vista.logica.ConfiguracionVistaLogica.ejecutarLoginVistaLogica(ConfiguracionVistaLogica.java:46)|co.com.codesa.socketcodesaoperadorexterno.vista.eventos.EventoFormaConfiguracion$1.run(EventoFormaConfiguracion.java:73)|java.lang.Thread.run(Thread.java:722)|
